src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
-
+server.port=8090
+spring.jpa.database=POSTGRESQL
+spring.datasource.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/student-info
+spring.datasource.username=postgres
+spring.datasource.password=password
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.properties.hibernate.format_sql=true
```

Comment:
Convert this as `application.yml` file. Only keep the following credentials in `application.properties`.
```
spring.datasource.username=postgres
spring.datasource.password=password
```

Reply:
Later, we can add the properties to git ignore and not be shown in github.